controller/controller.go

```diff
@@ -85,6 +86,9 @@ type Config struct {
 	// runtime objects the controller watches is performed. Defaults to
 	// DefaultResyncPeriod.
 	ResyncPeriod time.Duration
+	// (Optional) SentryDSN is the URL used to forward runtime errors to the sentry.io service.
```

Comment:
Golang best practise is to put the name of the documented identifier first. 

```suggestion
	// SentryDSN is the optional URL used to forward runtime errors to the sentry.io service.
```